config/config.yaml

```diff
@@ -9,12 +9,17 @@ default_container: "docker://gmsuppsala/somatic:develop"
 reference:
   design_bedfile: "/projects/wp1/nobackup/ngs/utveckling/Twist_DNA_DATA/bed/pool1_pool2_nochr_3c.sort.merged.padded20.hg19.210311.met.annotated.bed"
 
-cnvkit_call:
-  normal_reference: "/projects/wp1/nobackup/ngs/utveckling/Twist_DNA_DATA/bed/cnvkit_Twist_PoN.cnn"
-
-reference:
-  fasta: "reference/WASH7P.fna"
-  design_bedfile: "reference/twist_DNA_solid.chr1.annotated.bed"
-  
 germline_vcf:
   container: "docker://ensemblorg/ensembl-vep:release_99.0"
+
+cnvkit_call:
+  normal_reference: "/projects/wp1/nobackup/ngs/utveckling/Twist_DNA_DATA/PoN/cnvkit_Twist_PoN.cnn"
```

Comment:
should we maybe remove part of the path, the wp1 stuff. It's very specific to our environment and I guess we will not have that storage path as our final path.

Reply:
agreed, this should be an example on how to structure the `config.yaml` and not a specific setup.